### ISSUE TITLE
UI Opacity

### ISSUE THREAD

**Issue by andrey-zakharov** [fabmax/kool#37]
I'm trying to animate appeareance of UI stuff by smooth revealing from transparent state to full opacity.
I could not find out any easy way to make `UiSurface` semitransparent. I'd prefer to have some exposed shader's uniform as alphaMultiplier.
How could i make this?

**Comment by fabmax** [comment]
Yes some kind of global alpha multplier is still missing. All meshes that make up an UI are managed by `UiSurface` (`UiPrimitiveMesh`, `TextMesh` and `ImageMesh`), so it should be relatively easy to add an alphaMultiplier uniform to the corresponding shaders and add a property to `UiSurface` which then sets the shader uniforms.
